lib/handler/configurator/proxy.c

```diff
@@ -534,4 +554,7 @@ void h2o_proxy_register_configurator(h2o_globalconf_t *conf)
     h2o_configurator_define_command(&c->super, "proxy.max-buffer-size",
                                     H2O_CONFIGURATOR_FLAG_ALL_LEVELS | H2O_CONFIGURATOR_FLAG_EXPECT_SCALAR,
                                     on_config_max_buffer_size);
+    h2o_configurator_define_command(&c->super, "proxy.forward-early-hints",
```

Comment:
Please adjust how the configuration directives and possible other parts of the code that implies the feature to be specific to the reverse proxy.